air/src/options.rs

```diff
@@ -351,49 +351,56 @@ impl Deserializable for FieldExtension {
 #[derive(Debug, Clone, Copy, Eq, PartialEq)]
 pub struct PartitionOptions {
     num_partitions: u8,
-    min_partition_size: u8,
+    hash_rate: u8,
 }
```

Comment:
It may be good to add a brief explanation about how `num_partitions` and `hash_rate` are used to the doc comments for this struct.